qemu/tests/cfg/hotplug_block_resize.cfg

```diff
@@ -0,0 +1,52 @@
+- hotplug_block_resize:
+    only virtio_scsi, virtio_blk
+    virt_test_type = qemu
+    type = hotplug_block_resize
+    images += " stg0"
+    image_name_stg0 = images/storage0
+    blk_extra_params_stg0 = "serial=TARGET_DISK0"
+    remove_image_stg0 = yes
+    boot_drive_stg0 = no
+    force_create_image_stg0 = yes
+    kill_vm_on_error = yes
+    iozone_timeout= 3600
+    virtio_blk:
+        driver_name = viostor
+    virtio_scsi:
+        driver_name = vioscsi
+    Windows:
+        iozone_options = '-s 4k -s 64k -s 1m -b C:\{0}_stress_test -g {1} -I -i 0 -i 1 -f {0}:\iozone_test'
+    Linux:
+        iozone_options = '-s 4k -s 64k -s 1m -g {1} -I -i 0 -i 1 -f {0}/iozone_test'
+    variants:
+        - @default:
```

Comment:
IMHO, do not hide variants, it is easier to address the test

Reply:
Okay, I have updated it, thanks.